src/nvim/ex_cmds.c

```diff
@@ -4960,3 +4961,44 @@ void ex_oldfiles(exarg_T *eap)
     }
   }
 }
+
+void ex_trust(exarg_T *eap)
+{
+  char *p = skiptowhite(eap->arg);
+  size_t len = (size_t)(p - eap->arg);
+  char *arg1 = xcalloc(len + 1, sizeof(char));
+  memcpy(arg1, eap->arg, len);
```

Comment:
```suggestion
  char *arg1 = xmemdupz(eap->arg, len);
```

Reply:
Resolved in 27f98a78c168ff895c2134c27350ce57e7cdbcfe